app/controllers/animals_controller.rb

```diff
@@ -15,16 +15,16 @@ def create
     @animal = Animal.new(animal_params)
     @animal.user = current_user
     if @animal.save
-      redirect_to animal_path
+      redirect_to animal_path(@animal)
     else
       render 'new'
     end
   end
 
   private
 
-  def animal_params
-    params.require(:animal).permit(:name, :category, :species, :hourly_rate, :gender, :user, :place_of_origin, :description, :age)
+  def animal_params # I think user needs to be delete here, I'll do it on the next branch
+    params.require(:animal).permit(:name, :category, :species, :hourly_rate, :gender, :user, :place_of_origin, :description, :age, photos: [])
```

Comment:
I think :user needs to be :user_id here, as that's how it's listed under in the schema under the animals table.